src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -63,10 +63,10 @@ public void InitializeAppInfo(string installedLocation)
             InitPackageVersion(namespaces);
 
             const uint noAttribute = 0x80;
-            const Stgm exclusiveRead = Stgm.Read;
+            const STGM exclusiveRead = STGM.READ;
             var hResult = NativeMethods.SHCreateStreamOnFileEx(path, exclusiveRead, noAttribute, false, null, out IStream stream);
 
-            if (hResult == Hresult.Ok)
+            if (hResult == HRESULT.S_OK)
```

Comment:
why does IAppxManifestProperties have the forced `using HRESULT = Wox.Plugin.Common.Win32.HRESULT;` but then this doesn't?

Reply:
`using HRESULT = Wox.Plugin.Common.Win32.HRESULT;` is a [using alias](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#using-alias).
I thought it was clearer to specify that we're only `HRESULT` in the other file, but here we're using other stuff from `Wox.Plugin.Common.Win32` too. (like `STGM`)